core/utils_test.py

```diff
@@ -691,15 +691,15 @@ def test_get_time_in_millisecs(self) -> None:
         self.assertEqual(
             dt,
             datetime.datetime.fromtimestamp(
-                python_utils.divide(msecs, 1000.0))) # type: ignore[no-untyped-call]
+                (msecs // 1000.0)))
```

Comment:
Sorry for wrong isntructions before, but I looked into `past.utils.old_div` more and when some argument to it is of `float` type, you should use `/` instead of `//`. In some places it might even make sense to use `/` when both args are `int` as Sean pointed out above.

Reply:
yes, @vojtechjelinek I also did not notice that and those Backend errors are due to this only. 
But I fixed those.
Thanks for guiding....